samples/ipc_call_interface/src/math.c

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "az_ulib_action_api.h"
+#include "az_ulib_descriptor_api.h"
+#include "az_ulib_ipc_api.h"
+#include "math_tlb.h"
+#include "ucontract.h"
+#include "ulib_result.h"
+#include "ulog.h"
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * Concrete implementations of the math methods.
+ */
+static AZ_ULIB_RESULT sum_concrete(const void *const model_in, const void *model_out) {
+  AZ_UCONTRACT(
+      AZ_UCONTRACT_REQUIRE_NOT_NULL(model_in, AZ_ULIB_ILLEGAL_ARGUMENT_ERROR),
+      AZ_UCONTRACT_REQUIRE_NOT_NULL(model_out, AZ_ULIB_ILLEGAL_ARGUMENT_ERROR));
+
+  SUM_MODEL_IN *in = (SUM_MODEL_IN *)model_in;
+  SUM_MODEL_OUT *out = (SUM_MODEL_OUT *)model_out;
+
+  *out = in->a + in->b;
+
+  return AZ_ULIB_SUCCESS;
+}
+
+static AZ_ULIB_RESULT sub_concrete(const void *const model_in, const void *model_out) {
```

Comment:
>sub [](start = 22, length = 3)

nit: subtraction_concrete (I don't think sub is a popular acronym for subtraction)